test/amf1.yaml

```diff
@@ -20,4 +21,7 @@ spec:
     - name: vpc-internal
       interfaces:
         - n2
-  configRefs:
+  parametersRefs:
+          #    - apiVersion: workload.nephio.org/v1alpha1
```

Comment:
Are these omitted on purpose?

Reply:
configRefs are now parameterRefs but we can delete the commented part.